src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -449,6 +457,36 @@ public void ClearChildren()
 			InvalidateMeasure();
 		}
 
+		private void Cleanup()
+		{
+			if (this.GetParent() is UIElement originalParent)
+			{
+				originalParent.RemoveChild(this);
+				// Console.WriteLine($"Remove {HtmlId} from {originalParent.HtmlId}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)